python/brayns/core/common/bounds.py

```diff
@@ -36,6 +36,14 @@ def deserialize(message: dict) -> 'Bounds':
             max=Vector3(*message['max'])
         )
 
+    @classmethod
+    @property
+    def empty(cls) -> 'Bounds':
+        return cls(
+            min=Vector3.zero,
```

Comment:
Just a remark: If the bounds class ever adds functionality to build bounds from points/other bounds, this would be a problem.